app/src/main/java/io/apicurio/registry/metrics/health/liveness/ResponseErrorLivenessCheck.java

```diff
@@ -35,18 +35,18 @@ public class ResponseErrorLivenessCheck extends AbstractErrorCounterHealthCheck
      * to 60.
      * TODO report the absolute count as a metric?
      */
-    @ConfigProperty(name = "registry.metrics.ResponseErrorLivenessCheck.counterResetWindowDurationSec", defaultValue = "60")
+    @ConfigProperty(name = "apicurio.metrics.ResponseErrorLivenessCheck.counterResetWindowDurationSec", defaultValue = "60")
```

Comment:
Unify unit suffixes, `*Sec` vs `*.secs` vs `*.seconds`.

Reply:
Also we need to fix `counterResetWindowDurationSec` and any other camelCase properties.